examples/direct_fidelity_estimation.py

```diff
@@ -379,21 +381,25 @@ def direct_fidelity_estimation(
     n_qubits = len(qubits)
 
     clifford_circuit = True
-    clifford_state: Optional[cirq.CliffordState] = None
+    clifford_tableau: Optional[cirq.CliffordTableau] = None
     try:
-        clifford_state = cirq.CliffordState(qubit_map={qubits[i]: i for i in range(len(qubits))})
+        qubit_map = {qubits[i]: i for i in range(n_qubits)}
+        clifford_tableau = cirq.CliffordTableau(n_qubits)
```

Comment:
`qubit_map` also does not need to be inside the `try` block. If there is a `TypeError` here, it should loudly fail.

Reply:
Done.